docs/pages/versions/unversioned/sdk/background-task.mdx

```diff
@@ -0,0 +1,208 @@
+
+## Background tasks
+
+A background task is a deferrable unit of work that is performed in the background, outside the lifecycle of your app. This is useful for tasks that need to be executed when the app is inactive, such as syncing data with a server, fetching new content, or even checking if there are any [Expo Updates](updates.mdx).
+
+### When are background tasks run?
+
+The Expo Background Task API leverages each platform to execute tasks at the most optimal time for both the user and the device.
+
+This means that the task may not run immediately after it is scheduled, but it will run at some point in the future if the system decides so.
+
+A background task will only run if the battery has enough charge (or the device is plugged into power) and the network is available. Without these conditions, the task won't execute. The exact behavior will vary depending on the operating system.
+
+### When will they stop?
+
+The native platform APIs will stop running background tasks if the user explicitly kills the app (they will be resumed when the user restarts the app) or if the app is uninstalled. If the device is rebooted or the app is killed by the system, the background tasks will be resumed.
+
+> **Info** The non-deterministic behavior of background tasks is intentional, aimed at optimizing battery life and enhancing user experience. If a user force stops an app, the system interprets this as a signal not to run background tasks for that app until the user restarts it. On iOS an app is force stopped if the user swipes up on the app in the app switcher. On Android this behaviour depends on the device vendor. Some vendors have implementations where the app is force stopped when the user removes the app from the recent apps list.
+
+## Platform differences
+
+### Android&ensp;<PlatformTags platforms={['android']} />
+
+On Android, the [`WorkManager`](https://developer.android.com/topic/libraries/architecture/workmanager) API allows specifying a minimum interval for a task to run (at least 15 minutes). The task will execute sometime after the interval has passed, provided the specified conditions are met.
+
+### iOS&ensp;<PlatformTags platforms={['ios']} />
+
+On iOS, the [`BGTaskScheduler`](https://developer.apple.com/documentation/backgroundtasks/bgtaskscheduler) API decides the best time to launch your background task. The system will consider the battery level, the network availability, and the user's usage patterns to determine when to run the task.
+
+## Known limitations
+
+### iOS&ensp;<PlatformTags platforms={['ios']} />
+
+The [`Background Tasks`](https://developer.apple.com/documentation/backgroundtasks) API on iOS is not available on simulators, it is only available when running on a physical device.
```

Comment:
```suggestion
The [`Background Tasks`](https://developer.apple.com/documentation/backgroundtasks) API is unavailable on iOS simulators. It is only available when running on a physical device.
```

This can be concise.